<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>DisplayPrefsDialogUI</class>
 <widget class="QDialog" name="DisplayPrefsDialogUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Display Preferences</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>4</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="FontsGroup" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Fonts</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>5</number>
      </property>
      <property name="spacing" >
       <number>4</number>
      </property>
      <item row="1" column="0" >
       <widget class="QLabel" name="textLabel8" >
        <property name="text" >
         <string>Frame &amp; Error List:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="textLabel9" >
        <property name="text" >
         <string>Summary Pane:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="textLabel1" >
        <property name="text" >
         <string>Job &amp; Host List:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="JobFontEdit" />
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="FrameFontEdit" />
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="SummaryFontEdit" />
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="JobFontButton" >
        <property name="maximumSize" >
         <size>
          <width>40</width>
          <height>32767</height>
         </size>
        </property>
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QPushButton" name="FrameFontButton" >
        <property name="maximumSize" >
         <size>
          <width>40</width>
          <height>32767</height>
         </size>
        </property>
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QPushButton" name="SummaryFontButton" >
        <property name="maximumSize" >
         <size>
          <width>40</width>
          <height>32767</height>
         </size>
        </property>
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="Group" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>3</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Colors</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>6</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="mColorTree" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>211</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ApplyButton" >
       <property name="text" >
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OKButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>OKButton</sender>
   <signal>clicked()</signal>
   <receiver>DisplayPrefsDialogUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>323</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel" >
     <x>294</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>DisplayPrefsDialogUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>423</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel" >
     <x>397</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
